research/AGENT_CONTEXT_CONSOLIDATED_BRIEF.md

```
@@ -24,7 +24,7 @@ TensorZero uses **port 3000** for Docker-internal communication (confirmed in do
 | **Gateway Agent** | **8100** (internal) | — | services-catalog.md |
 | **Model Registry** | **8111** | 8110 (changed PR #845) | services-catalog.md |
 | **BoTZ VPN MCP** | **8110** | — | services-catalog.md (took 8110 after Model Registry moved) |
-| **Cipher Memory API** | **8096** (remapped from 3000) | — | services-catalog.md |
+| **Cipher Memory API** | **8105** (remapped from 3000) | — | services-catalog.md |
```

markdownlint-cli2 (0.22.0)

[warning] Table column count
Expected: 5; Actual: 4; Too few cells, row will be missing data

(MD056, table-column-count)